packages/@sanity/form-builder/src/inputs/PortableText/object/BlockObject.tsx

```diff
@@ -250,47 +250,47 @@ export function BlockObject(props: BlockObjectProps) {
   const hasErrors = errorMarkers.length > 0
   const hasWarnings = warningMarkers.length > 0
 
-  const markersToolTip = useMemo(
-    () =>
-      hasErrors || hasWarnings || (hasMarkers && renderCustomMarkers) ? (
-        <Tooltip
-          placement="top"
-          portal="editor"
-          content={
-            <TooltipBox padding={2}>
-              <Markers markers={markers} renderCustomMarkers={renderCustomMarkers} />
-            </TooltipBox>
-          }
-        >
-          <div>{blockPreview}</div>
-        </Tooltip>
-      ) : undefined,
-    [blockPreview, hasErrors, hasWarnings, hasMarkers, markers, renderCustomMarkers]
-  )
-
   const isImagePreview = type?.type?.name === 'image'
 
   const blockPath = useMemo(() => [{_key: block._key}], [block._key])
 
+  const tooltipEnabled = useMemo(
+    () => hasErrors || hasWarnings || (hasMarkers && renderCustomMarkers),
+    [hasErrors, hasMarkers, hasWarnings, renderCustomMarkers]
+  )
```

Comment:
Since [booleans are immutable](https://developer.mozilla.org/en-US/docs/Glossary/Primitive), the `useMemo` is not necessary.
```suggestion
  const tooltipEnabled = hasErrors || hasWarnings || Boolean(hasMarkers && renderCustomMarkers)
```